sycl/include/sycl/ext/intel/esimd/detail/simd_obj_impl.hpp

```diff
@@ -73,6 +73,16 @@ template <unsigned N> struct overaligned_tag {
   template <typename> static constexpr unsigned alignment = N;
 };
 
+/// \c dqword_element_aligned_tag type. Flag of this type should be used in load
+/// and store operations when memory address is aligned by simd object's element
+/// type or dword whatever is greater.
+struct dqword_element_aligned_tag {
```

Comment:
I think I have the last comment for this PR: Please move this class definition to detail namespace.
Using this class is rather a temporary internal solution used until we get approval for using element_aligned_tag for char/short loads.